src/lib/UI/Service/ContentTypeIconResolver.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\UI\Service;
+
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use Symfony\Component\Asset\Packages;
+
+class ContentTypeIconResolver
+{
+    /** @var \eZ\Publish\Core\MVC\ConfigResolverInterface */
+    private $configResolver;
+
+    /** @var \Symfony\Component\Asset\Packages */
+    private $packages;
+
+    /** @var string|null */
+    private $defaultThumbnail;
+
+    /**
+     * @param \eZ\Publish\Core\MVC\ConfigResolverInterface $configResolver
+     * @param \Symfony\Component\Asset\Packages $packages
+     * @param string|null $defaultThumbnail
```

Comment:
doc does not match constructor parameter.

Similiar tihing seems to be in `getContentTypeIcon` method

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/804/commits/c10c434f626e0ad965d58656ad8f971e051ed219